documentation/contributing/future_projects/index.markdown

```diff
@@ -171,4 +182,4 @@ For this, we need to identify and implement standard tasks at varying levels of
 - **Programming skills**: C++
 - **Difficulty**: Medium
 - **Potential mentors**: Mark Moll, David Lu!!
-- **Description**: The goal is to provide a standard way for integrating mobile base into move group that typically handles arms. While there's a non-standard way to incorporate holonomic drive that requires modifying your robot model ([example](https://groups.google.com/forum/#%21searchin/moveit-users/virtual$20base/moveit-users/G7vG7r23YzI/vpMUoa1hza4J)), customizing your robot package for such a common functionality is not ideal at all. We also extend to cover non-holonomic drive robots. [This is an FAQ](https://github.com/ros-planning/moveit/issues/375) among MoveIt users. Some initial work for this is currently underway for the [Hello Robot Stretch platform](https://hello-robot.com/product), but more work is needed to generalize to other types of mobile bases.
\ No newline at end of file
+- **Description**: The goal is to provide a standard way for integrating mobile base into move group that typically handles arms. While there's a non-standard way to incorporate holonomic drive that requires modifying your robot model ([example](https://groups.google.com/forum/#%21searchin/moveit-users/virtual$20base/moveit-users/G7vG7r23YzI/vpMUoa1hza4J)), customizing your robot package for such a common functionality is not ideal at all. We also extend to cover non-holonomic drive robots. [This is an FAQ](https://github.com/ros-planning/moveit/issues/375) among MoveIt users. Some initial work for this is currently underway for the [Hello Robot Stretch platform](https://hello-robot.com/product), but more work is needed to generalize to other types of mobile bases.
```

Comment:
I'm at a loss trying to find a diff in this line. Did you change the line ending?

Reply:
I did not change anything here.